sbin/build.sh

```diff
@@ -271,7 +271,7 @@ buildingTheRestOfTheConfigParameters()
   else
     addConfigureArg "--with-debug-level=" "release"
     addConfigureArg "--with-native-debug-symbols=" "none"
-    addConfigureArg "--enable-dtrace=" "auto"
```

Comment:
Shouldn't this be `"--enable-dtrace" ""`?

Reply:
Good point